lighthouse-cli/performance-experiment/experiment-database/index.js

```diff
@@ -0,0 +1,3 @@
+'use strict';
+
+module.exports = require('./database');
```

Comment:
Because I wanted to encapsulate the database (i.e. database and the temp file it generated will be treated as a whole). Although it will work perfectly as well if we just do `const ExperimentDatabase = require('./experiment-database/database');`